docs/v3-to-v4.md

```diff
@@ -2,14 +2,54 @@
 
 Version 4 of Node Redis is a major refactor. While we have tried to maintain backwards compatibility where possible, several interfaces have changed. Read this guide to understand the differences and how to implement version 4 in your application.
 
-## Breaking Changes
+## All the Breaking Changes
 
 See the [Change Log](../packages/client/CHANGELOG.md).
 
-## Promises
+### Promises
 
 Node Redis now uses native [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) by default for all functions.
 
+### `createClient`
+
+The configuration object passed to `createClient` has changed significantly with this release. See the [client configuration guide](./client-configuration.md) for details.
+
+### No Auto Connect
+
+In V4, the client will not be auto connected to the server, you need to run `.connect()` before any command, or you will receive error `ClientClosedError: The client is closed`.
+
+```typescript
+import { createClient } from 'redis';
+
+const client = createClient();
+
+await client.connect();
+
+await client.ping();
+```
+
+### No `message` event
+
+In V4, you don't need to add listener to `message` and `message_buffer` event, you can get the message directly in `subscribe`-like commands.
+
+The second agrument of commands is a callback, and will be triggered every time when there is a message published to the channel.
+
+The third agrument of commands is a boolean to set `bufferMode` (default `false`), if it's `true` you will receive buffer instead of string.
```

Comment:
```suggestion
The third argument to these commands is a boolean to set `bufferMode` (default `false`).  If it's set to `true` you will receive a buffer instead of a string.
```